CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+metrics-http-json.rb: Added the option to disable ssl cert verification
```

Comment:
these should go under an `### Added` header but I can fix that up for you this time. For future reference you can check out https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md